Common/GPU/Vulkan/thin3d_vulkan.cpp

```diff
@@ -1566,9 +1588,13 @@ void VKContext::BindFramebufferAsRenderTarget(Framebuffer *fbo, const RenderPass
 	curFramebuffer_ = fb;
 }
 
-void VKContext::BindFramebufferAsTexture(Framebuffer *fbo, int binding, FBChannel channelBit) {
+void VKContext::BindFramebufferAsTexture(Framebuffer *fbo, int binding, FBChannel channelBit, int layer) {
+	if (layer != ALL_LAYERS) {
+		layer = layer;
```

Comment:
Do we need a check here?  This isn't doing thing as-is anyway.

-[Unknown]

Reply:
That was just a silly way to add something to breakpoint on in debug. Removing.